js/src/components/campaign-conversion-notice/index.scss

```diff
@@ -27,6 +27,11 @@
 			.gla-campaign-conversion-status-notice__external_link {
 				margin-bottom: 0;
 			}
+
+			.gla-campaign-conversion-status-notice__external_link a {
+				font-weight: 600;
+				text-decoration: none;
+			}
```

Comment:
💅 
I think it could be simplified by removing `.gla-campaign-conversion-status-notice__external_link a {` and add the below lines under `.gla-campaign-conversion-status-notice__external_link {`

```css
.gla-campaign-conversion-status-notice__external_link {
  margin-bottom: 0;
  > a {
    font-weight: 600;
    text-decoration: none;
  }
}
```